content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,130 +1,108 @@
-scripts used to manage ML <abbr>artifacts</abbr> (training data, models, etc.)
-on cloud storage. DVC doesn't require special services, and works with
-on-premises storage (e.g. SSH, NAS) as well as any major cloud storage provider
-(Amazon S3, Microsoft Azure, Google Drive,
-[among others](/doc/command-reference/remote/add#supported-storage-types)).
-
-> For hands-on experience, we recommend following the
-> [versioning tutorial](/doc/use-cases/versioning-data-and-model-files).
-
-## DVC is not Git!
-
-DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to track
-data files and directories for versioning (among other purposes). They point to
-specific data contents in the <abbr>cache</abbr>, providing the ability to store
-multiple data versions out-of-the-box.
-
-Full-fledged
-[version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control)
-is left for Git and its hosting platforms (e.g. GitHub, GitLab) to handle. These
-are designed for source code management (SCM) however, and thus ill-equipped to
-support data science needs. That's where DVC comes in: with its built-in data
-<abbr>cache</abbr>, reproducible [pipelines](/doc/start/data-pipelines), among
-several other novel features (see [Get Started](/doc/start/) for a primer.)
-
-## Track data and models for versioning
-
-Let's say you have an empty <abbr>DVC repository</abbr> and put a dataset of
-images in the `images/` directory. You can start tracking it with `dvc add`.
-This generates a `.dvc` file, which can be committed to Git in order to save the
-project's version:
-
-```dvc
-$ ls images/
-0001.jpg 0002.jpg 0003.jpg 0004.jpg ...
-
-$ dvc add images/
-
-$ git add images.dvc .gitignore
-$ git commit -m "Track images dataset with DVC."
+# Versioning Data and Models
+
+Data science teams today face data management questions around versioning
+datasets, data artifacts, and machine learning models. How do we keep track of
+changes in data, code, and ML models? What's the best way to organize and store
+multiple versions of data and model files? How can data lifecycles be defined
+and enforced?
+
+![](/img/data_ver_complex.png) _Exponential complexity of DS projects_
+
+Let's see if DVC can help. Can we avoid ad hoc naming conventions for changed
+data files and directories? Yes: DVC stores the contents of datasets,
+intermediate results, and ML models as project snapshots are created
+([Git commits](<(https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository)>)).
+All these versions of the data are
+[cached](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory)
+outside of Git in an efficient way that prevents file duplication.
+
+<div style="display: flex; flex-flow: row wrap; margin: 0 -0.5rem;">
+<div style="flex: 1 0 50%; padding: 0.5rem;">
+
+**Before**: multiple filename versions — inefficient and error-prone
+
+```git
+ .
+ ├── data
```

Comment:
Hi @rogermparent 👋 Question!

As you may or may not see in https://dvc-landing-use-cases-v-6d9wm9.herokuapp.com/doc/use-cases/versioning-data-and-model-files (Ivan reported this is not rendered properly on his side), I'm trying to have a 2-column section in this use case:

![image](https://user-images.githubusercontent.com/1477535/98628694-c51b9200-22dc-11eb-92e2-af6f9d34b052.png)

I just copied some HTML from https://dvc.org/doc and it works on my side but not for every window width. What's the best way to do this? Can we perhaps generalize this layout and attach it to some tag?

p.s. I'm not 100% sure we'll keep it for this use case but it can surely be a useful tool in the engine, I think. Let's confirm with @shcheklein of course. Thanks

Reply:
> it works on my side but not for every window width

On this, of course we don't want 2 cols for slim windows, but I mean the current code breaks for some longer widths as well.

Reply:
>the current code breaks for some longer widths as well.

I'm actually not sure where the code is breaking here, unless you simply mean that the columns should collapse sooner than they do now so the columns have a smaller minimum width.

I that is the case, the simplest quick fix would be to change the `flex: 1 0 50%` styles to something like `flex: 1 0 325px`, changing the `flex-basis` to specify a preferred minimum width (325px is roughly around that the columns end up at with max screen width).

>  Can we perhaps generalize this layout and attach it to some tag?

Certainly! While nested elements like this can be tricky and have a bit of a clunky syntax, we can at least get a better syntax than raw copy+pasted HTML.

Reply:
> I'm actually not sure where the code is breaking here, unless you simply mean that the columns should collapse sooner

Hmmm, no. I swear it wasn't working on certain widths, but now it's OK!

@shcheklein could you confirm about this working for you and whether you think making a special component for this in our engine would be worth it?

Thanks